src/bootstrap/gerbil/compiler/driver__0.scm

```diff
@@ -791,7 +791,7 @@
                                        _libgerbil.so69300_)))))
                            (_gerbil-rpath69304_
                             (string-append
-                             '"-Wl,-rpath="
+			     (cond-expand (darwin "-Wl,-rpath,") (else "-Wl,-rpath="))
```

Comment:
revert this, the bootstrap compiler doesn't build any executables and it will disappear in the next regen.